File: code/data-investing.py

def obt_stock_hist(stock,country, from_d, to_d, as_json,order,interval):    
    import investpy
    data = investpy.stocks.get_stock_historical_data(stock=stock, country=country, from_date=from_d, to_date=to_d, as_json=as_json, order=order, interval=interval)
    return(data)
  

File: code/data-investing.R
library(reticulate)
source_python("code/data-investing.py")
df <- obt_stock_hist(
  stock ='bbva',
  country = 'spain',
  from_d = '01/01/2002',
  to_d = '01/01/2022',
  as_json = F,
  order = 'ascending',
  interval = 'Monthly') 

x <- select_emps[[1]] |>
  select(Date, Adjusted) |>
  mutate(Return_Ad = Delt(Adjusted)[,1]) |>
  na.omit() |>
  select(Date, Return_Ad)

